README.md

```diff
@@ -2,9 +2,9 @@
 
 This repository contains firmware and Wireshark extcap script that can be used with nRF52840 chip as an 802.15.4 sniffer.
 
-__Note:__ this project is experimental. There is __no Windows support__ in the current version.
```

Comment:
I meant to keep '__Note:__ this project is experimental.' and remove the other sentence.